tests/trx2junit.Tests/Internal/JUnitBuilderTests/Gh37.cs

```diff
@@ -0,0 +1,101 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Xml.Linq;
+using NUnit.Framework;
+
+namespace trx2junit.Tests.Internal.JUnitBuilderTests
+{
+
+    [TestFixture]
+    public class Gh37
```

Comment:
For this see https://github.com/cheng93/trx2junit/pull/1
After accepting this PR in your clone, it should flow back to this PR here.